lang/gst/Portfile

```diff
@@ -3,8 +3,8 @@
 PortSystem              1.0
 
 name                    gst
-version                 3.1
-revision                7
+version                 3.2.5
+#revision                0
```

Comment:
I would just nuke the revision line rather than commenting it out. That is our practice in these cases after all.

Reply:
Change made.